cape_privacy/policy/policy.py

```diff
@@ -94,7 +97,7 @@ def apply_policy(policy: data.Policy, df, inplace=False):
     return result_df
 
 
-def parse_policy(p: str):
+def parse_policy(p: Union[str, Dict[Any, Any]]):
     """Parses a policy yaml file.
 
     The passed in string can either be a path to a local file or
```

Comment:
Looks like this sentence is unfinished?

Reply:
Its continued on the line below. This needs to be updated though.